tests/README.md

```diff
@@ -154,11 +154,12 @@ It is a failure, and analogous to the standard `fail`.
 
 `[todo] out/zzz_test_todo.res -> not yet implemented`
 
-The produced result file contains the magic string "NOT YET IMPLEMENTED".
+The produced result file contains magic strings like `NOT YET IMPLEMENTED`.
```

Comment:
I find the `contains` too abstract. The string can contains anything of contain utill there is only one line containing `NOT YET ...`. What is the equality used? Can I use indentation before the line?